stacks/spica/src/storage/components/picker/picker.component.ts

```diff
@@ -46,7 +46,7 @@ export class PickerComponent implements OnInit {
   }
   uploadStorage(file: File): void {
     if (file) {
-      this.storage.upsertOne({name: file.name}, file).subscribe(
+      this.storage.update({name: file.name}, file).subscribe(
```

Comment:
Here are a few things I want to say

- First, why do we update storage files by their name?
- Also, the method name should be updateOne.